scheduler/config.py

```diff
@@ -0,0 +1,30 @@
+import os
+from dotenv import load_dotenv
+
+env = "production"
+TOPIC_EXCHANGE_NAME = "topic_exchange"
+# Binding Keys that scheduler has to listen to
+BINDING_KEYS = {
+    "MILESTONE_ADD":"events.projects.*.milestone.add",
+    "PROJECT_VERIFY":"events.projects.*.project.verify",
+    "TASK_ADD":"events.*.scheduler.task.add",
+    "OFFSET_RESERVE":"events.projects.*.offsets.reserve",
+    "OFFSET_COMMIT":"events.projects.*.offsets.commit",
+    "RATINGS_REWARD":"events.projects.*.ratings.reward",
```

Comment:
hmm from my understanding, penalizing does not fall under any of the use cases for the scheduler that's why i am not listening on it. If there is a gap in my understanding do let me know

Reply:
scheduler should listen to the penalise as well since as long as there are updates to the project, scheduler should know so it can remove the scheduled task pertaining to that resource so no unnecessary task is ran

Reply:
make sense. updated and commited